clubs/README.md

```diff
@@ -193,38 +188,35 @@ apps, websites, and games).
+Hack Club is like a continuous hackathon that happens twice a week at school.
 
 ## The First Meeting
 
-First, an important note: attendance tends to drop by approximately 25% after
-the first two meetings. However, members who return for the third meeting tend
-to stay for the rest of the year.
+At the first meeting, start by explaining that Hack Club allows people to learn
+and build things together over the course of the year, and then have members
+begin building their _first real thing™_ (a personal website), using our
+[Personal Website](../workshops/personal_website/README.md) workshop.
 
-Your goal is to reduce that attrition rate to 10% by making the first two
-meetings GREAT: exciting, engaging, and inspiring.
+At the second meeting, club members will complete their websites and demo their
+work at the end.
 
-To that end, here are two quick tips for the first meeting: 1. Bring food! Full
-bellies lead to fonder memories. Pastries or cookies from a local supermarket
-are easy, inexpensive, and effective. 2. Make an effort to chat with members who
-arrive early and leave late. Talk to them about their goals and interests, and
-get them personally pumped about what they'll do in Hack Club.
+Attendance tends to drop by approximately 25% after the first two meetings. On
+the bright side, members who return for the third meeting tend to return to
+subsequent meetings for the rest of the year. Your goal is to reduce that
+attrition rate by making the first two meetings exciting, engaging, and
+inspiring.
 
-At the highest level, your plan for the first meeting should be to explain what
-your Hack Club will do over the course of the year and then have students begin
-to begin building their _first real thing™_ (a personal website), using our
- [Personal Website](../workshops/personal_website/README.md)  workshop.
+Here are two quick tips to help that happen:
 
-At the second meeting, club members will complete their websites and then demo
-their work at the end of the meeting.
+1. Bring food! Full bellies lead to fonder memories. Pastries or cookies from a
+   local supermarket are easy, inexpensive, and effective.
+2. Make an effort to chat with members who arrive early and leave late. Talk to
+   them about their goals and interests, and get them personally pumped about
+   what they'll do in Hack Club.
 
-We recommend the format below for your first meeting. For each item in the
-schedule, there is a corresponding (and linked) section. We've also created a
-slideshow presentation to help you run your first meeting.
-[Click here][presentation] to go to it. Make a copy of the presentation to
-customize it for your club (to do this, go to `File > Make a copy...`). We'll
-refer to this presentation throughout the sections below.
+We recommend the following format for your first meeting. There are
+corresponding slides in the [Workshop Day presentation][workshop_day_template],
+and we've gone into more detail in the linked sections below.
```

Comment:
The slides for the first meeting are different than the slides for the workshop day. Does this remove references to the first meeting slides entirely?